script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -12,22 +12,22 @@ import FungibleToken from 0xee82856bf20e2aa6
 // Combined with an immutable owner account, i.e. an account without any
 // registered keys, this can also be used to enable a more "locked down" account
 // for doing sends/receives of FLOW tokens.
-pub contract FlowColdStorageProxy {
+access(all) contract FlowColdStorageProxy {
```

Comment:
I didn't see a reason to add any entitlements since all these functions should be possible to call from anyone

Reply:
That's my understanding as well